docs/src/standard_results/depth_surfaces.md

```diff
@@ -0,0 +1,44 @@
+# Modelled depth prediction surfaces
+
+This exports the modelled depth prediction surfaces from within RMS.
+These surfaces typically represent the final surface set generated during a structural
+modeling workflow (after well conditioning), and serve as the framework for constructing the grid.
+
+:::{table} Current
+:widths: auto
+:align: left
+
+| Field | Value |
+| --- | --- |
+| Version | NA |
+| Output | `share/results/maps/depth_predictions/surfacename.gri` |
+:::
+
+## Requirements
+
+- RMS
+- depth surfaces stored in a horizon folder within RMS
+
+The depth surfaces should be located within a horizon folder in RMS.
+The simplified export function will use the RMS API behind the scene to automatically
```

Comment:
Somewhat simplified suggestion:
This export function will automatically export all non-empty horizons from the provided folder.